Sources/Danger/Danger.swift

```diff
@@ -64,7 +68,7 @@ final class DangerRunner {
 // MARK: - Public Functions
 
 public func Danger() -> DangerDSL {
```

Comment:
<!--
  1 failure:  Function name sho...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/Danger/Danger.swift;
  Line: 70;
-->
- :no_entry_sign: Function name should start with a lowercase character: 'Danger()' (`identifier_name`)